service/src/main/java/org/whispersystems/textsecuregcm/controllers/AccountController.java

```diff
@@ -562,15 +562,15 @@ private CaptchaRequirement requiresCaptcha(String number, String transport, Stri
     try {
       rateLimiters.getSmsVoiceIpLimiter().validate(requester);
     } catch (RateLimitExceededException e) {
-      logger.info("Rate limited exceeded: " + transport + ", " + number + ", " + requester + " (" + forwardedFor + ")");
+      logger.info("Rate limited exceeded: {}, {}, {}, ({})", transport, number, requester, forwardedFor);
       rateLimitedHostMeter.mark();
       return new CaptchaRequirement(true, true);
     }
 
     try {
       rateLimiters.getSmsVoicePrefixLimiter().validate(Util.getNumberPrefix(number));
     } catch (RateLimitExceededException e) {
-      logger.info("Prefix rate limit exceeded: " + transport + ", " + number + ", (" + forwardedFor + ")");
+      logger.info("Prefix rate limit exceeded:{}, {}, ({})", transport, number, forwardedFor);
```

Comment:
There's a missing space, if you wanted it to be the same as the original.

Reply:
Done.